### PR TITLE
docs: add Wrappers API access security suggestion

### PR DESCRIPTION
**Pull request by burmecia**
## I have read the [CONTRIBUTING.md](https://github.com/supabase/supabase/blob/master/CONTRIBUTING.md) file.

YES

## What kind of change does this PR introduce?

This PR is to add API access security suggestion for Wrappers foreign data table.

## What is the current behavior?

Currently, the foreign table API access isn't properly suggested in the docs.

## What is the new behavior?

Suggest to use security definer function to access foreign table by API.

## Additional context

N/A

### CODE REVIEW

File: apps/docs/content/guides/database/extensions/wrappers/overview.mdx [hunk 1]
```diff
@@ -111,9 +111,84 @@ This process can be taxing on your database if you are moving large amounts of d
...
-
-If you want to expose any of the columns to your public API, create a [View](https://supabase.com/docs/guides/database/tables#views) in the `public` schema. You can also interact with your Foreign Tables inside a [Database Function](https://supabase.com/docs/guides/database/functions).
+Foreign Data Wrappers do not provide Row Level Security. We do not advise exposing them via your API. Wrappers should _always_ be stored in a private schema. For example, if you are connecting to your Stripe account, you should create a `stripe` schema to store all of your foreign tables inside. This schema should _not_ be added to the “Additional Schemas” setting in the API section.
+
+If you want to expose any of the foreign table columns to your public API, you can create a [Database Function with security definer](https://supabase.com/docs/guides/database/functions#security-definer-vs-invoker) in the `public` schema, and then you can interact with your foreign tables through API.
+
```
Comment (olirice) [inline apps/docs/content/guides/database/extensions/wrappers/overview.mdx:117]: a note here that they should add filters to apply whatever security rules they need inside the function would be good

File: apps/docs/content/guides/database/extensions/wrappers/overview.mdx [hunk 2]
```diff
@@ -111,9 +111,84 @@ This process can be taxing on your database if you are moving large amounts of d
...
+   begin
+     return query
+     select
+       t.id,
+       t.name,
+       t.active,
+       t.default_price,
+       t.description
+     from
+       stripe.stripe_products t;
```
Comment (olirice) [inline apps/docs/content/guides/database/extensions/wrappers/overview.mdx:163]: an example that contains a simple filter like filtering on `name` prefix would be a nice enhancement so users can see how to pass parameters through

File: apps/docs/content/guides/database/extensions/wrappers/overview.mdx (hunk 1 above)
Comment (burmecia) [reply apps/docs/content/guides/database/extensions/wrappers/overview.mdx:117]: okay added.

File: apps/docs/content/guides/database/extensions/wrappers/overview.mdx (hunk 2 above)
Comment (burmecia) [reply apps/docs/content/guides/database/extensions/wrappers/overview.mdx]: sure, add a function parameter in the sample code.

**Review by fsansalvadore — APPROVED** [review]

**Pull request merged**